packages/web/components/Dashboard/LatestContentTabs/Read.tsx

```diff
@@ -1,8 +1,51 @@
-import { Box, Text } from '@metafam/ds';
+import { Box, Heading, Link, LoadingState, Text } from '@metafam/ds';
 import React from 'react';
+import useSWR from 'swr';
 
-export const Read: React.FC = () => (
-  <Box>
-    <Text>A feed of news and events</Text>
-  </Box>
-);
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
+
+export const Read: React.FC = () => {
+  const { data, error } = useSWR('/api/feed', fetcher);
+
+  return (
+    <Box pt={4}>
+      {data && data.response ? (
+        data.response.items?.map(
+          (item: { title: string; description: string; link: string }) => (
+            <Box
+              key={item.title}
+              mt={4}
+              p={6}
+              backgroundColor="blackAlpha.500"
+              borderRadius="md"
+            >
+              <Heading size="xs" color="white" pb={4}>
+                {item.title}
+              </Heading>
+              <Text dangerouslySetInnerHTML={{ __html: item.description }} />
```

Comment:
Yup, I think this works, I would call our substack trusted content

Reply:
👍